Doc/library/uuid.rst

```diff
@@ -289,6 +289,25 @@ of the :attr:`~UUID.variant` attribute:
    Reserved for future definition.
 
 
+The :mod:`uuid` module defines the special Nil and Max UUID values:
+
+
+.. data:: NIL
+
+   A special form of UUID that is specified to have all 128 bits set to zero
+   according to :rfc:`RFC 9562, §5.9 <9562#section-5.9>`.
```

Comment:
Linking directly to the section is helpful, perhaps we only need to mention the RFC number here? If so, we could update the same thing for `uuid8` above.
```suggestion
   according to :rfc:`RFC 9562 <9562#section-5.9>`.
```